### PR TITLE
Doc/audio speech

### PR DESCRIPTION
**Pull request by yunlong-wang-cn**
update the source documentation for audio and command recognition, and some small changes.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,9 +61,9 @@ pip3 install wheel  # firstly install the wheel for further package building if
 pip3 install -r src/marimbabot/requirements.txt
 
 #install the precise engine
-wget https://github.com/MycroftAI/mycroft-precise/releases/download/v0.3.0/precise-all_0.3.0_x86_64.tar.gz
-tar -zxvf precise-all_0.3.0_x86_64.tar.gz -C src/marimbabot/marimbabot_speech/utils/kws/
-rm precise-all_0.3.0_x86_64.tar.gz
+wget https://github.com/MycroftAI/mycroft-precise/releases/download/v0.3.0/precise-engine_0.3.0_x86_64.tar.gz
```
Comment (Juphex) [inline README.md:64]: Did you test this?

File: marimbabot_audio/README.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
+# TAMS Master Project 2022/2023 - Audio Processing
```
Comment (Juphex) [inline marimbabot_audio/README.md:1]: You could also call it just "Audio" as in your speach README

File: marimbabot_audio/README.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+## 1. Motivation
+
+The robot's performance of playing the marimba need to be evaluated. Therefore, this submodule is designed to can evaluate the music produced by robot through the audio feedback. It can detect the western music note by the raw audio input, and visualize it in form of midi figure and  Constant-Q transform spectrum, also there is a final score for evaluate the final motion.  Moreover, it can synthesis the music from lilypond.
+
+## 2. Dependencies
+
+### 2.1 python packages
```
Comment (Juphex) [inline marimbabot_audio/README.md:9]: I do not think you need this, but I think it is alright

**Review by Juphex — APPROVED** [review]
Looks good.
If the new link works you could merge it

File: README.md (hunk 1 above)
Comment (yunlong-wang-cn) [reply README.md:64]: yes, this packages just work for inference.

**Pull request merged**